HelpAssistHer/help-assist-her
# Clear form on save

## Description
After a resource has been saved, we need to clear out the form so that when a new resource is loaded, none of the old data is included. This adds a button to clear the values. Note: this is not the most ideal way to clear the form, but with our upcoming enhancements, we will change this behavior.

## Test Plan
1. Load a resource
1. Change something
1. Click Save
1. Click 'Clear Values' and the values should all be cleared
